common/src/main/java/com/netflix/conductor/common/utils/TaskUtils.java

```diff
@@ -12,8 +12,20 @@
  */
 package com.netflix.conductor.common.utils;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class TaskUtils {
 
+    private static final Logger logger = LoggerFactory.getLogger(TaskUtils.class);
+
+    static {
```

Comment:
using a static block to log value of indexOnCommunicationFailure, introduces dependency on EnvUtils class to be loaded before TaskUtils. Is this fine or any other alternate way to log it ?

Reply:
no issues because of that.